extensions/wikia/Recirculation/RecirculationApiController.class.php

```diff
@@ -35,4 +35,38 @@ public function getCakeRelatedContent() {
 				'items' => (new CakeRelatedContentService())->getContentRelatedTo($target, $limit, $ignore),
 		]);
 	}
+
+	public function getAllPosts() {
+		$fandomDataService = new FandomDataService();
+		$fandom = [
+			'title' => wfMessage( 'recirculation-fandom-title' )->plain(),
+			'items' => $fandomDataService->getPosts( 'recent_popular', 12 )
+		];
+
+		$discussionsData = [];
+		if ( RecirculationHooks::canShowDiscussions() ) {
+			$discussionsDataService = new DiscussionsDataService();
+			$discussionsData = $discussionsDataService->getData();
+			$discussionsData['title'] = wfMessage( 'recirculation-discussion-title' )->plain();
+			$discussionsData['linkText'] = wfMessage( 'recirculation-discussion-link-text' )->plain();
+		}
+
+		$articleId = $this->request->getVal( 'articleId' );
+		$articles = $this->app->sendRequest( 'ArticlesApi', 'getTop', [
+			'abstract' => 0,
+			'expand' => 1,
+			'height' => 220,
+			'limit' => 8,
+			'namespaces' => 0,
+			'width' => 385,
```

Comment:
Yeah, it's for thumbnail sizes. These numbers specifically are the size required for large screens.